support/src/mock-clinical-trials/trials.js

```diff
@@ -76,6 +76,7 @@ const trialsGet = async (req, res, next) => {
  * @param {Express.Response} res
  * @param {Function} next
  */
+
```

Comment:
Revert changes here if you didn't actually make updates to this file

Reply:
gotcha